### DISCUSSION TITLE
Add an already created tiddler as a todo

### DISCUSSION THREAD

**Discussion by arunnbabu81** [NicolasPetton/Projectify#10] [Q&A] [answered]
Is there any way to add an already created tiddler as a todo directly from the project manager. Till now I used to do this by giving “todo” and “project title” tag to that particular tiddler. Is there any way to do it directly in the project manager itself.

My use case is here - I take notes from many articles. For that I create a tiddler each for each article. I found that providing a deadline will help in finish reading each article in time. So I have made  a project called “Take notes from articles” and I want to add each of these articles as a todo within that project and give a deadline for each article. Is there any way to do it directly in project manager rather than tagging each article with todo and project title tag.

**Comment by NicolasPetton** [comment]
@arunnbabu81 there's no dedicated interface to do that yet, though I think it's a great idea! 

I'm not how that would work in practive though. Do you have something in mind?

**Reply by arunnbabu81** [reply]
I am not aware of any way to do this since each tiddler's title must be unique. Shall wait for others to comment on this idea. Tagging with todo and project title will suffice for the time being.

**Comment by NicolasPetton** [comment, marked as answer]
@arunnbabu81 In the next release, you'll be able to drag&drop any tiddler link into a project todo-list or card to add it as a todo to that project.

If you setup your wiki to render tiddler titles as links, I think it would make it really easy for you to do what you want.

**Reply by NicolasPetton** [reply]
@arunnbabu81 v0.7.1 is now out, if you update you'll be able to add existing tiddlers as todos to projects.

**Reply by arunnbabu81** [reply]
Drag and drop works seamlessly in 0.7.1 once the tiddler links are enabled. Thanks for the update.
